install_bootstrap.md

```diff
@@ -0,0 +1,159 @@
+## What you will learn in this section
+
+This section will help you understand and install Bootstrap on your system.
+
+The first thing you need to do is access the getting started page on getBootstrap. Here you will notice three different download options depending on your level and skill set.
+
+If you’re completely new to Bootstrap, you’ll want to stick with the basic Bootstrap download. This contains compiled and minified CSS, JavaScript, and fonts. No docs or original source files are included.
+
+If you’re experienced using CSS pre-processors to extend the CSS language, there are two additional options available. The Source Code installation contains Source Less, JavaScript and font files. The SASS installation is Bootstrap ported from Less to Sass for easy inclusion in Rails, Compass, or Sass-only projects.
+
+In this tutorial we will cover Bootstrap for the beginner level for those who are new to Bootstrap. Hope you all have downloaded the Bootstrap from getBootstrap.
+
+### What is Next
+
+Once you have downloaded the Bootstrap you will get the following folder structure when you unpacke the compliled version of Bootstrap. Lets have a look,
+
+```
+bootstrap/
+├──css/
+|    ├── bootstrap.css
+|    ├── bootstrap.min.css
+|    ├── bootstrap-theme.css
+|    └── bootstrap-theme.min.css
+├──jss/
+|    ├── bootstrap.js
+|    └── bootstrap.min.js
+└── fonts/
+     ├── glyphicons-halflings-regular.eot
+     ├── glyphicons-halflings-regular.svg
+     ├── glyphicons-halflings-regular.ttf
+     └── glyphicons-halflings-regular.woff    
+```      
+This is the basic structure of the file you can also add the file in order to modify the standard css. We will cover it in the later part of this tutorial. How about having some magic and fun? We at Hello Django belief in the proverb 'All work and no play make Jack a dull boy'.
+So try the following steps:
+```
+Create a file 'index.html' in bootstrp folder
+Open 'index.html' in your editor.
+```
+
+Now lets copy the given bellow code and paste it in our file.
+
```

Comment:
I think we should start with most basic bootstrap template with minimal line of code instead of menubar version. This will help beginner understand easily.